app/models/resource.py

```diff
@@ -0,0 +1,30 @@
+from .. import db
+
+
+class Resource(db.Model):
+    __tablename__ = 'resources'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64))
+    address = db.relationship('Address', backref='resource',
+                              lazy='dynamic', uselist=False)
+    description = db.Column(db.Text)
+    website = db.Column(db.Text)
+    reviews = db.relationship('Review', backref='resource',
+                              lazy='dynamic')
+
+    def __repr__(self):
+        return '<Resource \'%s\'>' % self.name
+
+
+class Address(db.Model):
```

Comment:
Do we want to keep this model in here or move it to the location.py model file that @riyer15 is working on?